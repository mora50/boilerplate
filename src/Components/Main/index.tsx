const Main = () => (
  <main>
    <h1>REACT AVANÃ‡ADO</h1>
  </main>
);

export default Main;
