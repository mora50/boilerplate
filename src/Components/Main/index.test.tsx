import Main from ".";
import { screen, render } from "@testing-library/react";

describe("main", () => {
  it("render without bug", () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole("heading", { name: /react avan√ßado/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
