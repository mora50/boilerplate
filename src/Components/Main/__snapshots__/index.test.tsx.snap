// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main render without bug 1`] = `
<main>
  <h1>
    REACT AVANÃ‡ADO
  </h1>
</main>
`;
